pipelinewise/fastsync/mysql_to_snowflake.py

```diff
@@ -41,8 +41,8 @@ def tap_type_to_target_type(mysql_type, mysql_column_type):
         'char': 'VARCHAR',
         'varchar': 'VARCHAR',
         'binary': 'VARCHAR',
```

Comment:
This needs to be BINARY

Reply:
thank you, fixed by 8a510713d6c4cf61bdcc1d039ca9dac35d40dc16